source/standards/programming-languages.html.md.erb

```diff
@@ -47,7 +47,7 @@ appropriate. For example, the GOV.UK PaaS team uses TypeScript
 because they are used to working with a statically typed, compiled language,
 and they think the compilation and static-analysis tooling is better for
 their workflow. There's more information about TypeScript on the
-[Node.js][nodejs] page.
+[Node.js page](/manuals/programming-languages/nodejs/#transpiling).
```

Comment:
```suggestion
[Node.js page](/manuals/programming-languages/nodejs/).
```
Don't think we wanted the fragment link here?

Reply:
I put the fragment because this is the section that talks about TypeScript, but happy to remove if preferred.